src/library_dylink.js

```diff
@@ -315,8 +312,6 @@ var LibraryDylink = {
         } else if (typeof value == 'bigint') {
           entry.value = value;
 #endif
-        } else {
-          throw new Error(`bad export type for '${symName}': ${typeof value}`);
```

Comment:
I am guessing these changes are to do with handling a Memory export. But I see only removals. Where is the code that handles it?

Reply:
This should should just ignore exports such as memories and tables.  Its only interesting in handling globals and functions.